src/App.tsx

```diff
@@ -123,7 +122,15 @@ export const App: React.FC = () => {
                         window.open('https://www.eaton.com', '_blank');
                     }}
                 >
-                    <img src={EatonLogo} style={{ margin: '10px' }} alt={'Eaton Logo'} height={50} width={'auto'} />
+                    <img
+                        src={
+                            'https://raw.githubusercontent.com/pxblue/react-design-patterns/master/src/assets/EatonLogo.svg'
```

Comment:
What's the motivation here? Does CS not do well with local assets?

Reply:
It doesn't this time when I tried it.